engine/src/test/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStoreSpec.scala

```diff
@@ -244,22 +254,130 @@ class SqlWorkflowStoreSpec extends AnyFlatSpec with CromwellTimeoutSpec with Mat
       } yield ()).futureValue
     }
 
-    it should "select appropriately with the excludedGroups parameter" taggedAs DbmsTest in {
+    it should "select workflows from hog groups to start in round robin approach" taggedAs DbmsTest in {
+      // first submission of 50 workflows for hogGroup "Goldfinger"
+      for (_ <- 1 to 50) {
+        workflowStore.add(includedGroupSourceFilesCollection1)
+      }
+      Thread.sleep(5.seconds.toMillis)
+      // second submission of 50 workflows for hogGroup "Highlander"
+      for (_ <- 1 to 50) {
+        workflowStore.add(includedGroupSourceFilesCollection2)
+      }
+
+      val workflowsList = for (_ <- 1 to 10) yield {
+        (for {
+          // since both hog groups have 0 workflows running, the hog groups are sorted alphabetically and first one is picked
+          startableWorkflows1 <- workflowStore.fetchStartableWorkflows(5, "A08", 1.minute, Set.empty[String])
+          _ = startableWorkflows1.map(_.hogGroup.value).toSet.head should be("Goldfinger")
+          _ = updateWfToRunning(startableWorkflows1)
```

Comment:
Question for reviewers: Would it make sense to also assert that the workflow IDs returned in `startableWorkflows1` actually belong to the same hog group from submission time or would it be overkill? 
For example, something like
```
_ =  submissionResponses.map(_.id).toList should contain startableWorkflows1.map(_.id)
```